doc/defx.txt

```diff
@@ -784,6 +784,10 @@ OPTIONS							*defx-options*
 		"floating".
 
 		Default: "none"
+						*defx-option-show-parent*
+-show-parent
+		Show ../ parent directory by default.
```

Comment:
`by default` is not needed.

Reply:
`Show parent "../" directory in the buffer` is better for me.

Reply:
> `Show parent "../" directory in the buffer` is better for me.

get.